docs/guide/content/user_guide/downloading_files.md

```diff
@@ -56,7 +56,7 @@ At the top of the Downloads tab, a summary panel displays key information about
 The large circular indicator highlights the **percentage of completed downloads**, helping you quickly assess progress at a glance.
 
 
-#### Metadata Fields
+#### Displayed File Fields
```

Comment:
Not sure about the title. Metadata is mentioned in the `Project Detail Page – Downloads Tab` intro
What about: `Files Metadata`